data/src/main/java/camp/nextstep/edu/github/data/NetworkModule.kt

```diff
@@ -0,0 +1,46 @@
+package camp.nextstep.edu.github.data
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+import okhttp3.logging.HttpLoggingInterceptor
+
+@Module
+@InstallIn(SingletonComponent::class)
+class NetworkModule {
+
+    private val BASE_URL = "https://api.github.com"
+
+    @Provides
+    @Singleton
+    fun provideOkHttpClient(): OkHttpClient {
+        return OkHttpClient.Builder()
+            .addInterceptor(
+                HttpLoggingInterceptor().apply {
+                    level = HttpLoggingInterceptor.Level.BODY
+                }
+            )
+            .build()
+    }
+
+    @Provides
+    @Singleton
+    fun provideRetrofit(okHttpClient: OkHttpClient): Retrofit {
+        return Retrofit.Builder()
+            .client(okHttpClient)
+            .baseUrl(BASE_URL)
+            .addConverterFactory(GsonConverterFactory.create())
+            .build()
+    }
+
+    @Provides
+    @Singleton
+    internal fun provideGithubService(retrofit: Retrofit): ApiService {
```

Comment:
```suggestion
   fun provideGithubService(retrofit: Retrofit): ApiService {
```
NetworkModule에 대해서는 data 모듈에서만 접근하는 것이 의미가 있어요.
에러를 피하기 위해서 단순히 함수에 internal 접근자를 추가하신 것으로 보이는데, 클래스에 internal 접근자를 추가해보면 어떨까요?